log/handlers.ts

```diff
@@ -51,6 +52,42 @@ export class BaseHandler {
 }
 
 export class ConsoleHandler extends BaseHandler {
```

Comment:
It'd be better if one could enable/disable coloring. Check out `FileHandlerOptions` for example of additional handler config.

Reply:
I agree having that switch would be good - but maybe that should be done in the color module?

Reply:
Can you give an example?

My idea was:
`new ConsoleHandler("DEBUG", { colors: false })`

Reply:
@ry @bartlomieju done!